examples/purchaseTesterTypescript/app/screens/FooterPaywallScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import RevenueCatUI from 'react-native-purchases-ui';
+
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import RootStackParamList from '../RootStackParamList';
+
+type Props = NativeStackScreenProps<RootStackParamList, 'FooterPaywall'>;
+
+const FooterPaywallScreen: React.FC<Props> = () => {
+  return (
+    <RevenueCatUI.PaywallFooterContainerView style={{ backgroundColor: '#f8f8f8' }}>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nec
```

Comment:
LOL I guess we're keeping the lorem ipsum footer for the time being

Reply:
we can come back to it later

Reply:
Yup, need an improved sample app.